#
# Problem set 4 - new attempt
#

from problemSet4Utilities import *

print 'Success'
